Crashlytics/CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 # Unreleased
 - [added] Updated upload-symbols to 13.7 with VisionPro build phase support. (#12306)
+- [changed] Added support for Crashlytics to report information from Remote Config.
```

Comment:
As a an app developer, I don't know what this means. What kind of information is reported?

Reply:
We had similar changelog on Android, cc @danasilver for details.

Reply:
Yeah, this is the same changelog entry that we used for Android. The information is metadata about active Remote Config rollouts to aid with crash debugging.

Maybe we can say "Added support for Crashlytics to report metadata about Remote Config keys and values." since rollouts isn't available in the console yet and might be confusing to reference

Reply:
Change: https://github.com/firebase/firebase-ios-sdk/pull/12515